koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -439,6 +439,17 @@ def _write_incremental_logs(self, build_id, logs_dir):
             raise ContainerError("Build log finished but build still has not "
                                  "finished: %s." % build_response.status)
 
+    def _upload_user_warnings(self, logs_dir):
```

Comment:
upload where? This method is not uploading anything

Reply:
fixed